### DISCUSSION TITLE
Working directory does not match my file

### DISCUSSION THREAD

**Discussion by Redeltaz** [nvim-tree/nvim-tree.lua#2469] [Q&A]
### Description

When I'm calling neovim by adding a path parameter, nvim-tree don't show the good directory, it only show the directory of where I executed the command.
It seem very basic but I've checked a lot of issues and other website posts, everyone just suggest to add the `update_focused_file` section with `update_cwd`, and it work, but not for me.
My config is not new, I've used it for a year and everything was ok until I reinstall my computer today, it doesn't work anymore.

### Neovim version

```text
NVIM v0.9.4
Build type: Release
LuaJIT 2.1.1694285958
```

### Operating system and version

Linux 6.5.6-arch2-1 x86_64

### nvim-tree version

59e65d8

### Clean room replication

```lua
require'nvim-tree'.setup {
     update_focused_file = {
         enable = true,
         update_cwd = true,
         ignore_list = {}
     },
     filters = {
         dotfiles = false
     },
     git = {
         enable = true,
         ignore = false
     }
 }
```

### Steps to reproduce

1. go to your $HOME directory
2. `nvim Document/randomDirectory`

### Expected behavior

I want nvim-tree to show the directory of the path that I pass as parameter when I call nvim, for exemple if I execute `nvim /myDirectory` I want nvim-tree to show only what is on `/myDirectory`, not the root directory of were the command is executed

### Actual behavior

nvim-tree only show the directory of were the `nvim /yourPath` command is executed

**Comment by alex-courtis** [comment]
nvim-tree does not do any magic on startup, only acting upon changes at runtime.

Take a look at [Open At Startup](https://github.com/nvim-tree/nvim-tree.lua/wiki/Open-At-Startup) for some recipes you can use.

For directories, I use [Open For Directories And Change Neovim's Directory](https://github.com/nvim-tree/nvim-tree.lua/wiki/Open-At-Startup#open-for-directories-and-change-neovims-directory)

Converting this to a discussion. We can reopen a bug if an issue arises.

**Comment by Redeltaz** [comment]
strangely nothing of your suggestions worked. I had to add this line inside my nvim config : `vim.cmd('cd %:h')` so the working directory automatically change when I run nvim with a path argument.
It now work well but it's strange, I never had to do this before...

**Reply by alex-courtis** [reply]
> strangely nothing of your suggestions worked.

Which recipes didn't work? What happened?

> I had to add this line inside my nvim config : `vim.cmd('cd %:h')`

Did `vim.cmd.cd('%:h')` work? Sounds like it might be a substitution issue.
